htsget-search/src/htsget/mod.rs

```diff
@@ -70,6 +76,29 @@ impl From<StorageError> for HtsGetError {
   }
 }
 
+// TODO: See if there's a way to proxy the ParseErrors from noodles_core without repeating them here with our custom messages.
```

Comment:
I recall writing those errors here... not sure they are needed anymore at all? @CastilloDel can you remove this and see if it fails and/or refactor this piece?

Reply:
I'll take a look now!

Reply:
Yeah, they weren't being used. Away they go!